tests/python/frontend/pytorch/test_forward.py

```diff
@@ -466,6 +466,17 @@ def forward(self, *args):
     verify_model(Arange11().float().eval())
     verify_model(Arange12().float().eval())
 
+def test_forward_mesh_grid():
+    torch.set_grad_enabled(False)
+
+    class MeshGrid1(Module):
+        def forward(self, *args):
+            x = torch.tensor([1, 2, 3])
+            y = torch.tensor([4, 5, 6])
```

Comment:
Do we support parsing scalars aswell? eg: y = torch.tensor(6)

Reply:
Yes, added a test cases.